app/src/main/java/org/schabi/newpipe/views/AnimatedProgressBar.java

```diff
@@ -26,6 +30,23 @@ public AnimatedProgressBar(final Context context, final AttributeSet attrs,
         super(context, attrs, defStyleAttr);
     }
 
+    /**
+     * For video items in lists, progress bar for playback information should only display when
+     * option `Positions in lists` is enabled.
+     * @return whether `Positions in lists` is enabled
+     */
+    public Boolean getPositionsInListsEnabled() {
```

Comment:
This can be a primitive ``boolean``

Reply:
currently in some situations where progress bar information better refresh (for user experience) it won't refresh, I think there may be further improvement, like use listener to update. Previously I just calculate the boolean once but finally write the code that calculate it dynamically, as I think there's better be an approach to update its value. I'm noob in real-world programming and would be appreciated if you describe more about the benefit that just use a `boolean`.